Gems/EditorPythonBindings/Code/Source/PythonUtility.cpp

```diff
@@ -207,6 +211,10 @@ namespace EditorPythonBindings
                     parameter.m_typeId = behaviorArgument.m_typeId;
 
                     bool handled = 
+                        ConvertBehaviorParameterEnum<short>(obj, underlyingTypeId, parameter) ||
+                        ConvertBehaviorParameterEnum<unsigned short>(obj, underlyingTypeId, parameter) ||
```

Comment:
`short` and `unsigned short` are covered by `AZ::s16` and `AZ::u16` respectively as they are always the same type.
Only `long` and `unsigned long` needs to be covered